### PR TITLE
add number formatting and pricing data

### PR DESCRIPTION
**Pull request by wesbos**
Add number formatting - commas a nice to read large values.

Also added pricing data for some common models. 

Screenshot 2023-05-26 at 1 37 25 PM

### CODE REVIEW

File: src/Stat.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+}
+
+const formatter = new Intl.NumberFormat("en-US", {
+  style: "currency",
+  currency: "USD",
+});
+
+const pricePerToken = {
+  'gpt-3.5-turbo': 0.006 / 1000,
+  'gpt4': {
```
Comment (niieani) [inline src/Stat.tsx]: nit pick: missing dash between gpt and 4. Ideally, let's use official model names.

```suggestion
  'gpt-4': {
```

File: src/Stat.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    prompt: 0.03 / 1000,
+    completion: 0.06 / 1000,
+  },
+  'gpt4-32k': {
```
Comment (niieani) [inline src/Stat.tsx]:
```suggestion
  'gpt-4-32k': {
```

**Review by niieani — CHANGES_REQUESTED** [review]
Hey @wesbos! Thanks for this, looks like very useful additions.
Overall looks great, small remarks:
- The prices for gpt-4* and gpt-3.5-turbo should only be displayed when `cl100k_base` is the selected encoding. Other encodings are used by older models (e.g. `text-davinci-003`) which have different prices, so the cost would be inaccurate after the switch. I'd suggest to either add prices for other models, or just hide them when prices are unavailable. You can import the `modelToEncodingMap` const from [`gpt-tokenizer/mapping`](https://github.com/niieani/gpt-tokenizer/blob/774cf367d8b6737376437b9bd9866bc143e8b33a/src/mapping.ts#L14)
- As per our discussion on Twitter, `claude`* models use a different tokenization strategy. I'm happy to accept a PR to `gpt-tokenizer` repo to include the tokenization strategy for anthropic's models (could be ported from their python tokenizer), but for now, these would be inaccurate, so I would suggest removing them.

**Comment by wesbos** [comment]
thanks for the feedback - Ive implemented the requested changes, also added a description to each, and added the embeddings engine. Might be a bigger project to eventually show all pricing data - but this is handy enough for now!

**Review by niieani — APPROVED** [review]
Thanks!

**Pull request merged**